components/plurals/data/build.rs

```diff
@@ -0,0 +1,12 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+fn main() {
+    if std::env::var("CARGO_FEATURE_DATA").is_ok() {
```

Comment:
Thought: You probably don't need to check for `CARGO_FEATURE_DATA`; the `cfg` should be ignored if the feature is disabled.